examples/builder/builder.rs

```diff
@@ -9,9 +9,9 @@ use std::{collections::HashSet, time::Duration};
 #[allow(clippy::trivial_regex)]
 async fn main() -> Result<()> {
     // Excludes
-    let excludes = Some(RegexSet::new(&[r"example"]).unwrap());
+    let excludes = Some(RegexSet::new([r"example"]).unwrap());
```

Comment:
Note: these changes are unrelated to the feature, but I've added them to make clippy happy after upgrading to Rust 1.65.